pinot-controller/src/test/java/org/apache/pinot/controller/utils/SegmentMetadataMockUtils.java

```diff
@@ -81,15 +79,13 @@ public static SegmentMetadata mockSegmentMetadataWithPartitionInfo(String tableN
     if (columnName != null) {
       when(segmentMetadata.getColumnMetadataFor(columnName)).thenReturn(columnMetadata);
     }
-    when(segmentMetadata.getTableName()).thenReturn(tableName);
```

Comment:
these seem to be from segment metadata, not  segment zk metadata

Reply:
reverted